### PR TITLE
Alert and Button Added

### PR DESCRIPTION
**Pull request by nikhil-sharma-nks**
You can view the library at 
ALERT - https://silk-ui.netlify.app/components/alert.html
BUTTON - https://silk-ui.netlify.app/components/button.html
Changes made :-
1. Added Alert component to the library
(https://www.loom.com/share/2dc931e2875245548e95ade648af1b51)

2. Added Button component to the library 
(https://www.loom.com/share/f00f6235a2ab4ddeaf0679c02ff39abc)

### CODE REVIEW

File: components/alert.html [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
```
Comment (viru2001) [inline components/alert.html:12]: you can import this in your main CSS file

File: components/alert.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            style="
+              width: 100%;
+              height: 450px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
```
Comment (viru2001) [inline components/alert.html:143]: you can make class for this. try to avoid inline styling

File: styles/utilities.css [hunk 3]
```diff
@@ -73,17 +77,41 @@
...
   border-radius: 0 50px 50px 0;
 }
-.main-content .component-drawer .component-link:hover {
-  /* background-color: rgb(226, 226, 226);
-  color: var(--primary-color); */
+
+.main-content .component-drawer .component-link:hover,
+.main-content .component-drawer .current-component {
   background-color: var(--primary-color);
   color: #fff;
```
Comment (viru2001) [inline styles/utilities.css:84]: you can use variable for this

File: styles/utilities.css [hunk 4]
```diff
@@ -107,3 +135,56 @@
...
 }
+
+/* UTILITIES */
+
+.flex-col {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1rem;
+}
```
Comment (viru2001) [inline styles/utilities.css:146]: classname and properties mismatched

File: styles/variables.css [hunk 5]
```diff
@@ -6,8 +6,14 @@
   --font-semi-bold: 600;
   --font-bold: 700;
   --font-black: 900;
-  /* background: #d3dedc; */
 
+  /* COLORS */
   --primary-color: #6886c5;
-  --secondary-color: red;
+  --secondary-color: rgb(100, 237, 255);
+  --success: #4caf50;
```
Comment (viru2001) [inline styles/variables.css:13]: you should use only 1 color unit

**Review by viru2001 — CHANGES_REQUESTED** [review]

**Review by saifudinbadani — COMMENTED** [review]
add a floating button as well, other wise looks good to merge.

**Pull request merged**